lib/commands.js

```diff
@@ -273,10 +274,26 @@ var operations = module.exports.operations = {
       context.newTemplate = templateBody;
       context.oldTemplate = oldTemplate;
       context.oldParameters = oldParams;
+
       context.next();
     });
   },
 
+  getMasterConfig: function(context) {
+    if (context.overrides.masterConfig) {
+      var getMasterConfig = d3.queue();
```

Comment:
Since you've only got one async call to make here, using d3-queue doesn't really do anything for you.